dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml

```diff
@@ -414,8 +414,7 @@
                                     BorderThickness="{ThemeResource AutoSuggestListBorderThemeThickness}"
                                     BorderBrush="{ThemeResource AutoSuggestBoxSuggestionsListBorderBrush}"
                                     Background="{ThemeResource AutoSuggestBoxSuggestionsListBackground}"
-                                    contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                                    contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}">
+                                    CornerRadius="{ThemeResource OverlayCornerRadius}">
```

Comment:
In some cases we're gating setting CornerRadius on contract7Present, whereas here we're not - is that intentional?

Reply:
Yes. [Control.CornerRadius](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.cornerradius#additional-features-and-requirements) was newly add in api7, but on Border and Grid it's available since api v1.

Here we are setting CornerRadius on a Border, we were gating on contract7Present before because we were trying to bind the value to its template parent (which is AutoSuggestBox, and it doesn't have CornerRadius property until api7). Based on the design spec I attached in PR description, we actually don't want to bind the value to template parent, it should read value directly from OverlayCornerRadius resource, so we don't need conditional xaml any more.

Reply:
Got it, makes sense.  Thanks!